tools/real_time_graphing.py

```diff
@@ -0,0 +1,306 @@
+import logging
+from time import sleep, time
+
+import threading
+from multiprocessing import Queue
+
+import numpy as np
+from matplotlib import pyplot as plt, animation as animation
+
+try:
```

Comment:
get rid of try excepts for imports and use isort to sort all imports in the tools